client/css/latest.module.css

```diff
@@ -14,3 +14,18 @@
   padding-right: 16px;
 }
 
+.modalbutton {
+  font-size: 14px;
+  padding: 8px 24px;
+  border: 1px solid #a9afbc;
+  border-radius: 5px;
+  background: white;
+  color: #31363f;
+  font-family: Noto,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica Neue,Helvetica,Arial,sans-serif;
```

Comment:
You could put the font-family in a rule for the body if it will be used by most components.